src/test/groovy/io/jsonwebtoken/JwtParserTest.groovy

```diff
@@ -1,7 +1,7 @@
 /*
  * Copyright (C) 2014 jsonwebtoken.io
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
+ * Licensed under the Apache License, Version 2.0 (the "License")
```

Comment:
This is standard Apache License header - it shouldn't have been removed.  For example: https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/blob/master/ide-configuration/LICENSE-ASL-2.0-HEADER.txt